apps/next/app/api/deploy/[tag]/status/route.ts

```diff
@@ -46,12 +55,33 @@ const mockGetDeploymentStatus = async (deploymentTag: string): Promise<Deploymen
   });
 };
 
-async function* makeIterator() {
-  yield encoder.encode('Deployment in progress...')
-  await sleep(5000)
-  yield encoder.encode('Cleaning deployment files...')
-  await sleep(5000)
-  yield encoder.encode('Deployed...')
+async function* makeIterator(buildTag: string) {
+  yield encoder.encode('Planning build...');
+  const { stdout: stdoutJson } = await exec('nixpacks plan ./temp');
+  const plan = JSON.parse(stdoutJson) as NixpackPlan;
+  // Filtering out 'npm-9_x' because for some reason it's not listed on:
+  // https://search.nixos.org/packages
+  plan.phases.setup.nixPkgs =
+    plan.phases.setup.nixPkgs.filter(nixPackage => nixPackage !== 'npm-9_x');
+  writeFileSync('./temp/build-plan.json', JSON.stringify(plan, null, 2), "utf8");
+  await sleep(1000);
```

Comment:
why the sleep here?

Reply:
Some times the `yield`s happen in quick succession, causing the CLI to show output like the below. This is a quick (possibly temp) fix to allow some space between progress updates. Thought about spliting on the client but worried that I'd print the second statement prematurely.
```
[5] Checking deployment status...
[6] Planning build...Generating Dockerfile...  <== Two updates on one line.
[7] Building OCI image...
```